### PR TITLE
[BuildBot] Upgrade L0Loader version for Win.

### PR DESCRIPTION
**Pull request by rbegam**
Signed-off-by: rbegam <rehana.begam@intel.com>

### CODE REVIEW

File: sycl/plugins/level_zero/CMakeLists.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
   #      cmake files simplification
   include(ExternalProject)
   if (WIN32)
-    set(LEVEL_ZERO_LOADER_SOURCE_VERSION v1.0.16)
+    set(LEVEL_ZERO_LOADER_SOURCE_VERSION v1.2.3)
   else()
     set(LEVEL_ZERO_LOADER_SOURCE_VERSION v1.2.3)
   endif()
```
Comment (smaslov-intel) [inline sycl/plugins/level_zero/CMakeLists.txt:32]: Please remove if/else

Comment (rbegam) [reply sycl/plugins/level_zero/CMakeLists.txt:32]: In case we need diff versions for win/linux in future again?

Comment (smaslov-intel) [reply sycl/plugins/level_zero/CMakeLists.txt:32]: The if would be added then

Comment (rbegam) [reply sycl/plugins/level_zero/CMakeLists.txt]: done.

**Review by smaslov-intel — APPROVED** [review]
LGTM

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**